modules/backend/clinic/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(CLINIC_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/Clinic.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/room/Room.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/staff/doctor/Doctor.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/staff/Staff.cpp
+)
+
+add_library(clinic-lib STATIC ${CLINIC_SOURCES})
+
+target_compile_options(clinic-lib PRIVATE -Werror -Wall -Wextra -pedantic)
```

Comment:
What do you think about becoming more strict in production code?
For example those could be added:
` -Wunnused -Wshadow -Wold-style-cast -Wconversion -Wdangling-pointer`

Reply:
Good idea, I will add it.

Reply:
Done